libs/content/workspace/src/tab/markdown_editor/editor.rs

```diff
@@ -285,26 +268,15 @@ impl Editor {
                 },
             });
 
-            (true, true, true)
-        };
-        let appearance_updated = {
-            let capture_already_disabled = self
-                .appearance
-                .markdown_capture_disabled_for_cursor_paragraph;
-            self.appearance
-                .markdown_capture_disabled_for_cursor_paragraph = ui.input(|i| i.modifiers.command); // command key disables capture for current paragraph
```

Comment:
cleanup; capture is already disabled for the current paragraph regardless of command key